### PR TITLE
User2 branch

### PR DESCRIPTION
**Pull request by jkumar712**

### CODE REVIEW

File: app/adapters/details.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import ApplicationAdapter from './application';
+
+export default ApplicationAdapter.extend({
```
Comment (dkdrk) [inline app/adapters/details.js]: remove this file from the branch and then commit

File: app/adapters/application.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import DS from 'ember-data';
+
+export default DS.JSONAPIAdapter.extend({
+});
```
Comment (dkdrk) [inline app/adapters/application.js]: remove this file from the branch and then commit

**Comment by jkumar712** [comment]
i deleted adapter and also push a file edit

File: app/adapters/details.js (hunk 1 above)
Comment (jkumar712) [inline app/adapters/details.js]: yes i removed

**Pull request merged**